### PR TITLE
OSDOCS-12919: Documented the 4.16.27 z-stream release notes

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.16

Issue:
[OSDOCS-12919](https://issues.redhat.com/browse/OSDOCS-12919)

Link to docs preview:
* [4.16.27](https://86261--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-27_release-notes)

### CODE REVIEW

**Review by GroceryBoyJr — COMMENTED** [review]
Two small nits, sir.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -3338,6 +3338,41 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, you could not remove a finally pipeline task from the *edit Pipeline* form if you created a pipeline with only one finally task. With this release, you can remove the finally task from the *edit Pipeline* form and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-45229[*OCPBUGS-45229*])
+
+* Previously, the installation program did not validate the maximum transmission unit (MTU) for a custom IPv6 network on {op-system-first}. If you specified a low value for the MTU, installation of the cluster would fail. With this release, the minimum MTU value for IPv6 networks is set to `1380`, where `1280` is the minimum MTU for IPv6 and `100` is the OVN-Kubernetes encapsulation overhead. With this release, the installation program now validates the MTU for a custom IPv6 network on {op-system-first} (link:https://issues.redhat.com/browse/OCPBUGS-41813[*OCPBUGS-41813*])
+
+* Previously, when the `Display Admission Webhook` warning implementation presented issues with some incorrect code. With this update, the unnecessary warning message has been removed. (link:https://issues.redhat.com/browse/OCPBUGS-43750[*OCPBUGS-43750*])
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:3367]: :heavy_check_mark:

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -3338,6 +3338,41 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, you could not remove a finally pipeline task from the *edit Pipeline* form if you created a pipeline with only one finally task. With this release, you can remove the finally task from the *edit Pipeline* form and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-45229[*OCPBUGS-45229*])
+
+* Previously, the installation program did not validate the maximum transmission unit (MTU) for a custom IPv6 network on {op-system-first}. If you specified a low value for the MTU, installation of the cluster would fail. With this release, the minimum MTU value for IPv6 networks is set to `1380`, where `1280` is the minimum MTU for IPv6 and `100` is the OVN-Kubernetes encapsulation overhead. With this release, the installation program now validates the MTU for a custom IPv6 network on {op-system-first} (link:https://issues.redhat.com/browse/OCPBUGS-41813[*OCPBUGS-41813*])
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:3365]: The dangers of copying an approved note from another release. I'll ask the SME. Thanks!

[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc]: OK. Updated to:

> With this release, the minimum MTU value for IPv6 networks is set to `1380` octets, where `1280` octets is the minimum MTU for the IPv6 protocol and the remaining `100` octets is for the OVN-Kubernetes encapsulation overhead.

Which is a much clearer statement. Thank you for spotting this!

**Pull request merged**